docker/clickhouse/init_schema.sql

```diff
@@ -256,3 +256,26 @@ ORDER BY _time;
 INSERT INTO default.test_barchart
 SELECT now() - INTERVAL number HOUR AS _time, map ('rocketStage', concat('stage', toString(number % 4)), 'rp_prescreenStep', '(prescreen)') AS user_metadata_map, rand() % 100 AS alloc_cost, concat('cluster', toString(number%4)) AS cluster, multiIf(cluster='cluster2', 'RESOURCE_SLEEP', cluster='cluster3', 'INTERACTIVE', concat('RESOURCE', rand() % 4)) AS hpcod_resource_name
 FROM numbers(24);
+
+-- Drop the table if it exists
```

Comment:
remove this comments or use `/* comment */`

sometime -- have weird side effects